lib/text.js

```diff
@@ -0,0 +1,89 @@
+let { PptxUnitHelper } = require('./helpers/unit-helper');
+let { Slide } = require('./slide');
+
+class Text {
+    constructor(args) {
+        Object.assign(this, args);
+
+        this.content = args.content;
+        this.powerPointFactory = args.powerPointFactory;
+        this.properties = this.content['p:spPr'][0];
+        this.parentContainer = args.parentContainer;
+    }
+
+    // TODO: these are the _exact_ same functions in the Image object; move them to a common class or inherit...
+    x(val) {
```

Comment:
agreed would love if this was in a helper class or we extended from it some how.

Reply:
Done.